playbooks/roles/mattermost/defaults/main.yml

```diff
@@ -4,7 +4,9 @@ MATTERMOST_HOSTNAME: "{{ inventory_hostname }}"
 MATTERMOST_OPS_EMAIL: ops@exmaple.com
 
 MATTERMOST_VERSION: 5.8.0
-MATTERMOST_UNLINK_VERSION: 0.3.0
+# UNLINK is the autolink plugin which converts JIRA tickets to clickable URL
+# https://github.com/mattermost/mattermost-plugin-autolink/releases/
+MATTERMOST_UNLINK_VERSION: 1.0.1
```

Comment:
@SSPJ, the latest version is `1.1.0` and I installed that version.

Reply:
@lgp171188 Right, this PR isn't for merging; I just pushed it so that you would have something to look at. I should have made that clear in the description, sorry!